tests/gold_tests/bigobj/bigobj.test.py

```diff
@@ -0,0 +1,129 @@
+'''
+Test client-side communication when GETting a very large object.
+'''
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+Test.Summary = '''
+Test client-side communication when GETting a very large object.
+'''
+
+Test.SkipUnless(
+    Condition.HasCurlFeature('http2'),
+    Condition.HasProgram('nc', 'nc (netcat) command needed for this test'),
+    Condition.HasProgram('cc', 'cc (C compiler) command needed for this test')
```

Comment:
Back in `6c1a90f4f209a02cf1345ac4c20e2e67c4c58db1`, it was decided not to check for things like nc.

And in the other tests that build custom c code, we don't test for cc.

Reply:
OK changed.